src/screens/NetworkSettings/NetworkSettingsFlowStack.js

```diff
@@ -5,11 +5,11 @@ import { NetworkSettingsDislaimerNav, NetworkSettingsDislaimerScreen } from './N
 
 export const NetworkSettingsFlowNav = Symbol('NetworkSettingsFlowStack').toString();
 
-export const NetworkSettingsFlowStack = ({ navigation }) => {
+export const NetworkSettingsFlowStack = () => {
   const FlowStack = createStackNavigator();
   return (
     <FlowStack.Navigator
-      initialRouteName={CustomNetworkSettingsNav}
+      initialRouteName={NetworkSettingsDislaimerNav}
```

Comment:
This is an error you have already fixed in other PRs, is it missing a rebase?

Reply:
Answered above.